src/nvim/api/options.c

```diff
@@ -68,8 +68,8 @@ static int validate_option_value_args(Dict(option) *opts, char *name, OptIndex *
     return FAIL;
   });
 
-  VALIDATE((!HAS_KEY_X(opts, scope) || !HAS_KEY_X(opts, buf)), "%s",
-           "cannot use both 'scope' and 'buf'", {
+  VALIDATE(!(HAS_KEY_X(opts, scope) && *scope == OPT_GLOBAL && HAS_KEY_X(opts, buf)), "%s",
+           "cannot use both 'scope=global' and 'buf'", {
```

Comment:
Other error messages don't use this kind of quoting for `key=val`

Reply:
I did a quick sweep of other `VALIDATE` usages but couldn't find one that specifically complained about wrong value. I changed it, if it's wrong please lemme know how it should look like.